src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -122,6 +125,8 @@ function cloneUpdateQueue(alt : Fiber, fiber : Fiber) : UpdateQueue | null {
   altQueue.first = sourceQueue.first;
   altQueue.last = sourceQueue.last;
   altQueue.hasForceUpdate = sourceQueue.hasForceUpdate;
+  altQueue.callbackList = sourceQueue.callbackList;
+  altQueue.isProcessing = sourceQueue.isProcessing;
```

Comment:
Why are we setting `isProcessing` here, but didn’t set before? Is this fixing a bug?